scorep/instrumenters/scorep_profile.py

```diff
@@ -1,9 +1,8 @@
 __all__ = ['ScorepProfile']
 
 import sys
-import inspect
-import os.path
-import scorep.instrumenters.base_instrumenter as base_instrumenter
+from scorep.instrumenters.base_instrumenter import get_module_name, get_file_name
+from scorep.instrumenters.scorep_instrumenter_base import ScorepInstrumenterBase
```

Comment:
Please see the above comment.

Reply:
See above. How does e.g. `scorep_instrumenter_base.ScorepInstrumenterBase` provide any benefit? I find that way harder to read